activiti-spring-identity/src/main/java/org/activiti/spring/identity/ActivitiUserGroupManagerImpl.java

```diff
@@ -0,0 +1,35 @@
+package org.activiti.spring.identity;
+
+import org.activiti.runtime.api.identity.UserGroupManager;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class ActivitiUserGroupManagerImpl implements UserGroupManager {
+
+    @Autowired
```

Comment:
Prefer constructor injection instead of field injection.

Reply:
👍 please modify it